packages/core/src/components/checkbox/checkbox.css

```diff
@@ -133,6 +133,10 @@
   padding-left: calc(var(--size) + var(--label-padding));
   padding-top: calc((var(--size) - (var(--label-font-size) * var(--lineheight-m))) / 2);
   position: relative;
+
+  &.hds-checkbox__no-label {
```

Comment:
I believe nesting classes does not work in css, or at least it should not 😀 So should move this outside this selector.

Reply:
I pressure we are using BEM? In that case I would interpret this as a modifier instead of a block:
`.hds-checkbox--no-label { ...`

Reply:
Ah could really be that I named it wrongly 👍 fixing all...

Reply:
And actually nesting classes does work and starts to be supported -> https://caniuse.com/css-nesting ping @mikkojamG But yeah maybe I'll make it the old-fashion-way then.

Reply:
I'm old school like that.. 🙂

Reply:
Now this should be corrected as old-school :D